tests/app/helpers.py

```diff
@@ -86,6 +86,20 @@
     "SQE2a": ["as a prime contractor, using third parties (subcontractors) to provide some services"]
 }
 
+EMPTY_G7_DRAFT = {
```

Comment:
I went with making it a function in the end - couldn't see any immediate benefit to making it a py.test fixture right now.